.github/workflows/pr_ci_backend.yaml

```diff
@@ -82,4 +82,4 @@ jobs:
       - name: Run pytest - Unit Tests
         if: always()
         run: |
-          pytest ./backend --cov=./backend --cov-report=term-missing --cov-fail-under=70 --cov-config=./backend/pyproject.toml -vv
+          pytest ./backend --cov=./backend --cov-report=term-missing --cov-fail-under=75 --cov-config=./backend/pyproject.toml -vv
```

Comment:
Updating it to 75 as we're at 80.6, so maybe we can move it to 80 once we're at say 83/84 😊